generate/writeToJsonFile.js

```diff
@@ -0,0 +1,82 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
```

Comment:
If I understand this code correctly, you're reading the existing file and only updating the changed bits. I would rather just have this override the original file with all the new data.